src/app/body/all-products/filters/filters.component.html

```diff
@@ -1,43 +1,79 @@
 <div class="filters_container">
   <div class="sort">
     <span class="openSans">Sort by</span>
-    <select name="Select">
-        <option selected disabled hidden>Select</option>
-        <option>Price</option>
-        <option>Freshness</option>
-    </select>
+    <label>
+      <select [(ngModel)]="selectedValue" (change)="sort(selectedValue)">
+          <option selected disabled hidden>Select</option>
+          <option
+            *ngFor="let o of sortOptions"
+            [ngValue]="o"
+          >{{o.label}}</option>
+      </select>
+    </label>
   </div>
 
-  <div class="filter">
-    <div class="filter_title">Categories</div>
-    <ul class="filter_items">
-      <app-category-filter></app-category-filter>
-      <app-category-filter></app-category-filter>
-      <app-category-filter></app-category-filter>
-      <app-category-filter></app-category-filter>
-    </ul>
-  </div>
+  <form id="form" novalidate [formGroup]="form" (change)="filter()">
+    <div class="filter">
+      <div class="filter_title">Categories</div>
+      <ul class="filter_items">
+        <li *ngFor="let category of categories">
+          <label class="filter_category">
+            <input type="radio" value="{{category}}" name="categories" formControlName="categories" />
```

Comment:
`value="{{category}}"   ->  [value]="category"`

Reply:
Change it everywhere.

Reply:
fixed